src/initial-parameters.ts

```diff
@@ -32,6 +36,11 @@ export async function determineInitialParameters(
           project.rootPackage.validatedManifest.name.replace('/', '__'),
         )
       : path.resolve(cwd, inputs.tempDirectory);
+  const parsedTodayTimestamp =
```

Comment:
This is moved from `monorepo-workflow-operations`, where it was done on the fly.